docs/user_guide/1_getting_started_hydromt/index.rst

```diff
@@ -0,0 +1,15 @@
+Getting started with HydroMT
+================================
+
+The core functionalities of HydroMT, such as the command-line interface (CLI), data catalog management, and region definitions, are documented in the main HydroMT documentation.
+These topics are shared across all HydroMT model plugins (including HydroMT-Wflow) and are maintained in one central location to ensure consistency.
+Refer to the links below for the official and up-to-date information.
+
+.. toctree::
+   :hidden:
+   :caption: HydroMT Core
+   :maxdepth: 2
+
+   CLI <https://deltares.github.io/hydromt/stable/guides/user_guide/hydromt_cli.html>
```

Comment:
We might update this with the refactoring of core docs. Might be good to also get the python example and not just CLI. Anyway we'll need to review the links to hydromt core as quite a few pages will have moved locations so we might need another PR to review links and then also review this point. Can you open the issue so that we do not forget? Will need to be done this sprint too as it's part of this issue and the core docs issue but we will need another PR I think.